### ISSUE TITLE
ios app

### ISSUE THREAD

**Issue by diiirt** [Speek-App/Speek#17]
Will there ever by an ios app?  I would like to give this a go but without an ios app it's isn't super useful for me.  If there is an ios app will it be able to recieve push notifications?  I recall reading somewhere that this was difficult to do in ios.  Thanks, keep up the good work!

**Comment by fabionolasco** [comment]
First, thank you for the Speek.network!
Any news on a possible version for iOS?
Is it on the back log?
Estimated time of delivery?

**Comment by fabionolasco** [comment]
Makes sense. We will need to get creative on this one... maybe using something like "installable" PWA with (a) Browser Notification or (b) Push Notification. While we wait for v1.7.0, maybe it could be a nice thing to put the word out to Developer and Hacker communities, including Universities, to request ideas. I can forward this thread to my students if you want to use this GitHub issue page to discuss it.

**Comment by AlexZigante** [comment]
just as a brainstorming idea:
how about switching to ionic.io and using this plugin for tor?
https://github.com/Start9Labs/capacitor-tor
